### PR TITLE
for terrains a batir only, display price per square meter

### PR DESCRIPTION
**Pull request by hugolpz**
First attempt edit. Review needed.

### CODE REVIEW

**Comment by hugolpz** [comment]
First, €/m² would be interesting for all lands with no building on it. It's the current PR.


----
Second, more advanced and complex, not in this PR.
With : 
- Price of a house sale (already have it)
- Calculation of the area average price by square meter (€/m²) for terrains a battir
It is possible to estimate the price of a land under a house, and therefor by substraction, the value of the house as total and per square meter.

This provide relevant estimates of the goods on a land.

File: static/js/component.boite-accordeon.js [hunk 1]
```diff
@@ -30,6 +30,12 @@ Vue.component('boite-accordeon', {
 							:valeur="formatterNombre(terrain['surface_terrain']) + ' m²'"
 							icone="fa fa-tree"
 							:texte="terrain['nature_culture'] + (terrain['nature_culture_speciale'] != 'None' ? ' / ' + terrain['nature_culture_speciale'] : '')">
+						</boite>
+						<boite <!-- for terrains a batir, display price per square meter -->
```
Comment (edouard-lopez) [inline static/js/component.boite-accordeon.js]: Comment in a tag? I reckon it won't work

File: static/js/component.boite-accordeon.js [hunk 2]
```diff
@@ -30,6 +30,12 @@ Vue.component('boite-accordeon', {
...
+							v-for="terrain in mutation.terrains"
+							:valeur="formatterNombre(mutation.infos[0]['valeur_fonciere'] / terrain['surface_terrain']) + ' €/m²'"
+							icone="fa fa-tree"
+							:texte="terrain['nature_culture'] + (terrain['nature_culture_speciale'] != 'None' && terrain['nature_culture'] != 'sols' && terrain['nature_culture_speciale'] != 'sols' ? ' / ' + terrain['nature_culture_speciale'] : '')">
```
Comment (edouard-lopez) [inline static/js/component.boite-accordeon.js]: The last condition look weird:

	terrain['nature_culture_speciale'] != 'sols' ? ' / ' + terrain['nature_culture_speciale'] : ''

**Comment by hugolpz** [comment]
I will close this.

**Pull request closed without merging**